dashboard/app/controllers/api/v1/regional_partners_controller.rb

```diff
@@ -11,4 +11,13 @@ def index
       render json: regional_partner_mapping, include: :regional_partner
     end
   end
+
+  # GET /api/v1/regional-partner/for_user
+  def for_user
+    if current_user.permission? UserPermission::WORKSHOP_ADMIN
+      render json: RegionalPartner.all.pluck(:id, :name)
+    else
+      render json: current_user.regional_partners.pluck(:name, :id)
```

Comment:
Any reason the order of these params are swapped? How about using json instead of tuples so the order doesn't matter.